CHANGELOG.md

```diff
@@ -2,8 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [19.1.0](https://github.com/ngneat/spectator/compare/v19.0.0...v19.1.0) (2024-09-12)
 
+## v.Next
```

Comment:
please revert, it should be updated automatically upon release

Reply:
Ok.